### PR TITLE
Add cloud dashboard to changelog

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Adds the cloud dashboard to the changelog
- Fixes broken links in changelog

### Important Notes
The specific issue to link to for the "cloud dashboard" feature can be changed.
I think adding all relevant commits to the changelog would be unnecessary clutter - especially as the cloud dashboard is still being changed/getting new features relatively frequently.
(Also, it shouldn't be relevant news for end users as they are almost certainly not using the cloud dashboard yet.)
Of course, once the cloud dashboard is officially released, each feature PR should get its own changelog entry.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [x] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Review by PabloBuchu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -156,11 +156,14 @@
   <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>t</kbd>/<kbd>r</kbd>.
 - [Fixed a bug where selecting a nested breadcrumb would cause the order of
   breadcrumbs to change incorrectly.][6617]
+- [Cloud Dashboard, which supersedes the startup screen][6279]. Features also
+  added in various other PRs.
```
Comment (wdanilo) [inline CHANGELOG.md]: This should be a user-friendly description. People form outside of the company should understand what changed, so such descriptions should be more wordy and provide more explanation of what really changed from user-perspective.

**Review by wdanilo — APPROVED** [review]
Perfect, than ks so much @somebody1234

**Comment by PabloBuchu** [comment]
@somebody1234 please solve conflicts so I can merge :)

**Comment by somebody1234** [comment]
@PabloBuchu merged with develop.

**Pull request merged**